FEATURE_MATRIX.md

```diff
@@ -1,64 +1,76 @@
+
+| | C# | Python | Java | Notes |
+|---|---|---|---|---|
+| TextMemorySkill                   | ✅ | ✅ | 🔄 | |
+| ConversationSummarySkill          | ✅ | ✅ | ❌ | |
+| FileIOSkill                       | ✅ | ✅ | ❌ | |
+| HttpSkill                         | ✅ | ✅ | ❌ | |
+| MathSkill                         | ✅ | ✅ | ❌ | |
+| TextSkill                         | ✅ | ✅ | 🔄 | |
+| TimeSkill                         | ✅ | ✅ | 🔄 | |
 
 ## Planning
-| | C# | Python | Notes |
-|---|---|---|---|
-| Plan | ✅ | ❌ | |
-| SequentialPlanner | ✅ | ❌ | |
+
+| | C# | Python | Java | Notes |
+|---|---|---|---|---|
+| Plan              | ✅ | ❌ | ❌ | |
+| SequentialPlanner | ✅ | ❌ | ❌ | |
 
 ## Connectors and Skill Libraries
-| | C# | Python | Notes |
-|---|---|---|---|
-| Qdrant (Memory)                   | ✅ | ❌ | Vector optimized |
-| ChromaDb (Memory)                 | ❌ | 🔄 | |
-| Milvus (Memory)                   | ❌ | ❌ | Vector optimized |
-| Pinecone (Memory)                 | ✅ | ❌ | Vector optimized |
-| Weaviate (Memory)                 | ❌ | ❌ | Vector optimized |
-| CosmosDB (Memory)                 | ✅ | ❌ | CosmosDB is not optimized for vector storage |
-| Sqlite (Memory)                   | ✅ | ❌ | Sqlite is not optimized for vector storage |
-| Postgres (Memory)                 | ✅ | ❌ | Vector optimized (required the [pgvector](https://github.com/pgvector/pgvector) extension) |
-| Azure Cognitive Search            | ❌ | ❌ | |
-| MsGraph                           | ✅ | ❌ | Contains connectors for OneDrive, Outlook, ToDos, and Organization Hierarchies |
-| Document Skills                   | ✅ | ❌ | Currently only supports Word documents |
-| OpenAPI                           | ✅ | ❌ | |
-| Web Skills                        | ✅ | ❌ | |
-
-# Design Choices
+
+| | C# | Python | Java | Notes |
+|---|---|---|---|---|
+| Qdrant (Memory)                   | ✅ | ❌ | ❌ | Vector optimized |
+| ChromaDb (Memory)                 | ❌ | 🔄 | ❌ | |
+| Milvus (Memory)                   | ❌ | ❌ | ❌ | Vector optimized |
+| Pinecone (Memory)                 | ✅ | ❌ | ❌ | Vector optimized |
+| Weaviate (Memory)                 | ❌ | ❌ | ❌ | Vector optimized |
+| CosmosDB (Memory)                 | ✅ | ❌ | ❌ | CosmosDB is not optimized for vector storage |
+| Sqlite (Memory)                   | ✅ | ❌ | ❌ | Sqlite is not optimized for vector storage |
+| Postgres (Memory)                 | ✅ | ❌ | ❌ | Vector optimized (required the [pgvector](https://github.com/pgvector/pgvector) extension) |
+| Azure Cognitive Search            | ❌ | ❌ | ❌ | |
+| MsGraph                           | ✅ | ❌ | ❌ | Contains connectors for OneDrive, Outlook, ToDos, and Organization Hierarchies |
+| Document Skills                   | ✅ | ❌ | ❌ | Currently only supports Word documents |
+| OpenAPI                           | ✅ | ❌ | ❌ | |
+| Web Skills                        | ✅ | ❌ | ❌ | |
+
+## Design Choices
 
 The overall architecture of the core kernel is consistent across Python and C#,
```

Comment:
@brunoborges can we include Java in this sentence? Or if not explain why.

Reply:
Fixed in [8e560df](https://github.com/microsoft/semantic-kernel/pull/1189/commits/8e560dffb56e668b8cfee26d049fe8ff71347230)